lib/asyncify.js

```diff
@@ -48,7 +48,7 @@ import initialParams from './internal/initialParams';
  *     }
  * ], callback);
  *
- * // es6 example
+ * // es2017 example
```

Comment:
Would it be useful to add a note here that `async` functions are supported out of the box (when not transpiled)?